code/game/objects/items/devices/camera_bug.dm

```diff
@@ -25,6 +25,7 @@
 	integrated_console = new(src)
 	integrated_console.parent = src
 	integrated_console.network = list("SS13", "camera_bug[UID()]")
+	GLOB.restricted_camera_networks += "camera_bug[UID()]"
 
 /obj/item/camera_bug/Destroy()
 	QDEL_NULL(integrated_console)
```

Comment:
Headcoders are in favor of making this change, should be a quick one.
```suggestion
	GLOB.restricted_camera_networks -= "camera_bug[UID()]"
	QDEL_NULL(integrated_console)
```

Reply:
99% sure this means that if this happens, and the camera bug item is deleted, it means that the AI will be able to view the  hidden camera network, Wouldn't it? If headcoders prefer that, thats fine, I just want to make sure all the side effects are properly communicated.

Reply:
I see what you mean, I'll bring that up

Reply:
Yeah, it seems like we don't need to worry about this as per discussions